test-integration/helpers/window-position.coffee

```diff
@@ -0,0 +1,12 @@
+windowPosition = (test) ->
```

Comment:
It looks like other helpers use the class syntax while this one just relies on closures, thoughts on which method we should use in general?